octobot_trading/modes/scripted_trading_mode/abstract_scripted_trading_mode.py

```diff
@@ -108,19 +108,36 @@ async def reload_scripts(self):
 
     async def start_over_database(self):
         await modes_util.clear_plotting_cache(self)
-        symbol_db = databases.RunDatabasesProvider.instance().get_symbol_db(self.bot_id,
-                                                                            self.exchange_manager.exchange_name,
-                                                                            self.symbol)
+        symbol_db = databases.RunDatabasesProvider.instance().get_symbol_db(
+            self.bot_id, self.exchange_manager.exchange_name, self.symbol
+        )
         symbol_db.set_initialized_flags(False)
         for producer in self.producers:
-            for time_frame, call_args in producer.last_call_by_timeframe.items():
```

Comment:
we can't use `producer.last_call_by_timeframe` ?

Reply:
the data was not fully available when manual execution gets triggered